src/libs/runtime-utils.js

```diff
@@ -351,9 +359,45 @@ export const isCurrentGalaxyDiskDetaching = apps => {
 export const getGalaxyCostTextChildren = (app, appDataDisks) => {
   const dataDisk = getCurrentAttachedDataDisk(app, appDataDisks)
   return app ?
-    [getComputeStatusForDisplay(app.status), dataDisk ? ` (${Utils.formatUSD(getGalaxyCost(app, dataDisk))} / hr)` : ``] : ['None']
+    [getComputeStatusForDisplay(app.status), dataDisk ? ` ${Utils.formatUSD(getGalaxyCost(app, dataDisk))}/hr` : ``] : ['']
 }
 
+// TODO: multiple runtime: this is a good example of how the code should look when multiple runtimes are allowed, over a tool-centric approach
+export const getCostDisplayForTool = (app, currentRuntime, currentRuntimeTool, toolLabel) => {
```

Comment:
as was discussed in slack.  If these methods are only used by one visual component (especially since they mention "display" in their name), lets move them over to live next to the visual component as whatever-component-utils.js.  And if this lets you move over your unit tests and mocks, great!